services/storage/src/simcore_service_storage/handlers.py

```diff
@@ -315,6 +338,30 @@ async def download_file(request: web.Request):
         return {"error": None, "data": {"link": link}}
 
 
+@routes.get(f"/{api_vtag}/locations/{{location_id}}/files/{{fileId}}/s3/link")  # type: ignore
+async def get_s3_link(request: web.Request) -> Dict[str, Any]:
+    params, query, body = await extract_and_validate(request)
+
+    assert params, "params %s" % params  # nosec
+    assert query, "query %s" % query  # nosec
+    assert not body, "body %s" % body  # nosec
+
+    with handle_storage_errors():
+        location_id = params["location_id"]
+        user_id = query["user_id"]
+        file_uuid = urllib.parse.unquote_plus(params["fileId"])
+
+        if int(location_id) != SIMCORE_S3_ID:
+            raise web.HTTPForbidden(
```

Comment:
Why forbidden here?
*please* read https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403

The HTTP 403 Forbidden response status code indicates that the server understands the request but refuses to authorize it.